README.md

```diff
@@ -67,6 +67,27 @@ func main(){
 }
 ```
 
+### Specify a Region and/or Edge
+
+```go
+package main
+import (
+	"github.com/twilio/twilio-go/twilio"
+	"os"
+)
+
+func main(){
+    accountSid := os.Getenv("TWILIO_ACCOUNT_SID")
```

Comment:
nit: tabs vs spaces, please run this snippit through `gofmt` or I think the spaces just need a conversion to tabs.